java/plugins/jdi-elements-plugin/src/test/java/ru/sbtqa/tag/pagefactory/pages/jdielements/StepDefs.java

```diff
@@ -3,11 +3,12 @@
 import cucumber.api.java.Before;
 import ru.sbtqa.tag.pagefactory.JDIUtils;
 import ru.sbtqa.tag.pagefactory.PageFactory;
+import ru.sbtqa.tag.pagefactory.drivers.TagWebDriver;
 
 public class StepDefs {
     
     @Before
     public void initJDI() {
-        JDIUtils.setJDIConfig(() -> PageFactory.getDriver());
+        JDIUtils.setJDIConfig(() -> TagWebDriver.getDriver());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)